src/Libraries/RevitNodes/Elements/AdaptiveComponent.cs

```diff
@@ -127,8 +316,7 @@ public List<Point> Locations
         // without breaking replication guides
 
         [IsVisibleInDynamoLibrary(false)]
-        [AllowRankReduction]
-        public static AdaptiveComponent[] ByParametersOnFace(double[][][] uvs, ElementFaceReference faceReference, FamilyType familyType)
```

Comment:
Also would be a good idea to add the `Obsolete` attribute with an appropriate message